### PR TITLE
Tom follow-ups after 3.12.6 and 3.17.0 releases

### PR DESCRIPTION
**Pull request by Lex-2008**
Otherwise, he wasn't able to decyper tag and crashed.

----

#

### CODE REVIEW

File: tom/tag.py [hunk 1]
```diff
@@ -63,6 +63,9 @@ def get_next_build_tag(self, old_tag, branch):
                 raise TagParsingException('Could not split tag [%s] into parts' % old_tag)
         else:
             # 3.15.0 -> 3.16.0-build1 or 3.15.1-build1, depending on branch
+            # But first, strip release number from version
+            # 3.12.5-2 -> 3.12.5
+            old_tag = re.sub('-.*','',old_tag)
```
Comment (vpodzime) [inline tom/tag.py:68]: Is this expected to only remove `-[0-9]+`? If so, make the regex more specific, please.

**Review by vpodzime — COMMENTED** [review]
Looks good to me otherwise.

**Review by olehermanse — CHANGES_REQUESTED** [review]
@Lex-2008 LGTM, please address comment by @vpodzime.

File: tom/tag.py [hunk 2]
```diff
@@ -63,6 +63,9 @@ def get_next_build_tag(self, old_tag, branch):
                 raise TagParsingException('Could not split tag [%s] into parts' % old_tag)
         else:
             # 3.15.0 -> 3.16.0-build1 or 3.15.1-build1, depending on branch
+            # But first, strip release number from version
+            # 3.12.5-2 -> 3.12.5
+            old_tag = re.sub('-[0-9]+','',old_tag)
```
Comment (vpodzime) [inline tom/tag.py:68]: Would `-[0-9]+$` work to make sure we only strip the end of the string?

**Review by vpodzime — APPROVED** [review]
Looks good to me either way.

**Review by cf-bottom — APPROVED** [review]
I trust @vpodzime, approved!

File: tom/tag.py (hunk 2 above)
Comment (Lex-2008) [reply tom/tag.py:68]: well, the thing is that code that follows (`get_next_version`) expects a string of three dot-separated numbers. So proper way is to strip only the proper stuff (`-[0-9]+$`), and fail with a nice message when met with something unexpected.

**Pull request merged**